test/modules/json/data/deck-props.json

```diff
@@ -4,7 +4,7 @@
     "latitude": 37.8,
     "zoom": 12
   },
-  "controller": "MapController",
+  "controller": "@@!MapController",
```

Comment:
Why is this needed? `controller: true` would work with any type of view.

Reply:
This is just a fixture, used to test the use of the `constants` lookup configured in `test/modules/json/json-configuration-for-deck.js` on line 17.

Reply:
I'll find a more practical example, since the user would never modify this directly